index.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable lines-between-class-members */
+/* eslint-disable no-undef */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-unused-vars */
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] The purpose of linters is to write maintainable code and maintain a standard across teams. Disabling linters defeats that purpose. Kindly remove these disabling and solve errors that come up.

  If you are stuck you can use google to find the solution for the errors. For example, the last error `max-classes-per-file`, requires that only one class should be in a file - which is a convention. You can separate classes into different files and export them, then import them accordingly.

Reply:
Can you please help me with that? @DammyShittu

Reply:
Please it is in the requirement that we can disable these linters.
![day2](https://user-images.githubusercontent.com/109290913/205926507-084db983-a887-46b4-b68b-4014378fff19.png)